autoload/neomake/makers/ft/crystal.vim

```diff
@@ -0,0 +1,16 @@
+function! neomake#makers#ft#crystal#EnabledMakers()
```

Comment:
I think a lot of the files have it at the bottom, and we do not have a policy for that.
IMHO we should not have them in general, but rather a .editorconfig / .lvimrc file.